specs/core/0_beacon-chain.md

```diff
@@ -1413,10 +1413,9 @@ def process_crosslinks(state: BeaconState) -> None:
 
 ```python
 def get_base_reward(state: BeaconState, index: ValidatorIndex) -> Gwei:
-    adjusted_quotient = integer_squareroot(get_total_active_balance(state)) // BASE_REWARD_QUOTIENT
-    if adjusted_quotient == 0:
-        return 0
-    return state.validator_registry[index].effective_balance // adjusted_quotient // BASE_REWARDS_PER_EPOCH
+    total_balance = get_total_active_balance(state)
+    effective_balance = state.validator_registry[index].effective_balance
+    return effective_balance * BASE_REWARD_QUOTIENT // integer_squareroot(total_balance) // BASE_REWARDS_PER_EPOCH
```

Comment:
Slight preference for the alternative ordering given the name `BASE_REWARD_QUOTIENT`. Otherwise, we'd need to change `BASE_REWARD_QUOTIENT` to something else.

Reply:
I'm a fan of renaming that anyway, the name has always seemed off.

Reply:
Renamed to `BASE_REWARD_FACTOR`.